onboard/src/pages/subsections/backgroundInformation/EmploymentDesired.jsx

```diff
@@ -1,7 +1,253 @@
-import React from "react";
+import React, { useState } from "react";
+import Radio from "@mui/material/Radio";
+import RadioGroup from "@mui/material/RadioGroup";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import FormControl from "@mui/material/FormControl";
+import FormLabel from "@mui/material/FormLabel";
+import Grid from "@mui/material/Grid";
+import InputField from "../../../components/inputs/InputField";
+import Box from "@mui/material/Box";
+import { FormHelperText } from "@mui/material";
 
 function EmploymentDesired() {
-  return <></>;
+  const [position, setPosition] = useState("");
+  const [startDate, setStartDate] = useState("");
+  const [desiredSalary, setDesiredSalary] = useState("");
+  const [employmentStatus, setEmploymentStatus] = useState("");
+  const [contactCurrentEmployer, setContactCurrentEmployer] = useState("");
+  const [employedAtOCI, setEmployedAtOCI] = useState("");
+  const [nonCompeteStatus, setNonCompeteStatus] = useState("");
+  const [expirationDate, setExpirationDate] = useState("");
+
+  const [nonCompeteStatusHelper, setNonCompeteStatusHelper] = useState(
+    "Please select a valid option"
+  );
+  const [employmentStatusHelper, setEmploymentStatusHelper] = useState(
+    "Please select a valid option"
+  );
+  const [contactCurrentEmployerHelper, setContactCurrentEmployerHelper] =
+    useState("Please select a valid option");
+  const [employedAtOCIHelper, setEmployedAtOCIHelper] = useState(
+    "Please select a valid option"
+  );
+
+  function handleChange(event) {
+    const e = event;
+    const val = e.target.value;
+    const name = e.target.name;
+
+    console.log(val);
```

Comment:
We don't want to leave console logs in code when we are done with it. If it is a very important console, comment it out. Otherwise, delete.